content/en/docs/User guides/monitoring-your-jobs.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Monitoring your jobs"
+linkTitle: "Monitoring your jobs"
+date: 2019-05-09
+weight: 2
+description: >
+  Monitor your OpenCue rendering jobs from CueGUI
+---
+
+This guide describes how to monitor your OpenCue jobs from the CueGUI app
+and configure plugins for viewing job details.
+
+After you submit a job to OpenCue, you can monitor the progress of the
+individual frames and layers in the job from the CueGUI app on your
+workstation. CueGUI supports the following *views* to monitor and manage
+job status:
+
+*   *Cuetopia* includes plugins for monitoring jobs and individual job
+    details.
+*   *Cuecommaner* includes plugins for monitoring your OpenCue system.
+    System administrators typically run the Cuecommander view to monitor
+    and manage OpenCue infrastructure, such as rendering hosts.
+*   *Other* includes plugins for viewing logs and attributes.
+
+This guide explains how to use the Cuetopia view to monitor jobs. To
+learn more about the features available in CueGUI, see
+[CueGUI reference](/docs/reference/cuegui-reference/).
+
+## Before you begin
+
+Make sure you are familiar with the steps for
+[submitting jobs](/docs/user-guides/submitting-jobs/).
+
+You also need to configure the Cuetopia view.
+
+### Configuring Cuetopia
+
+1.  Start CueGUI.
+
+    {{% alert title="Note" color="info"%}}The instructions for running
+    Cuetopia vary depending on the installation method. To learn more, see
+    [Installing CueGUI](/docs/getting-started/installing-cuegui/)
+    or contact your OpenCue admin.{{% /alert %}}
+
+    After CueGUI loads for the first time, you might see one or more blank
+    windows, as illustrated by the following screenshot:
```

Comment:
The blank window shouldn't be a problem anymore, hopefully. There was a bug that was preventing the default window layout from being read in if an existing one wasn't found. We didn't know that wasn't the intended behavior!

Reply:
With that in mind I suspect you might want to rework this first section a bit. Sorry about that.

Reply:
No worries, I should have tried to test with a fresh install but I'm finding it tricky to be certain that I'm resetting my test environment. Is there a way to reset CueGUI to default so I can see what the windows look like on first launch? (e.g. delete a config file) I'm assuming the window layout is local to the workstation and not stored server-side?

Alternatively, are you able to share a screenshot or describe what the initial layout (list of plugins) looks like?

Reply:
If you remove `~/.config/.cuecommander/config.ini` that should do it. On next startup CueGUI will generate some log messages like:

```
WARNING   Main       Not found: /Users/cipriano/.config/.cuecommander/config.ini
Copying:   cuegui/config/cuecommander.ini
```

Which show that it is initializing the default config for you.